<?xml version="1.0"?>
<launch>

    <node pkg="usb_cam" type="usb_cam_node" name="cam_0" output="screen">
        <param name="video_device" value="/dev/video2" />
        <param name="camera_frame_id" value="/camera/cam_0" />
        <param name="camera_name" value="${NAME}" />
        <param name="camera_info_url" value="package://ieee2016_vision/calibrations/${NAME}.yaml" />
        <param name="pixel_format" value="yuyv" />
        <param name="image_width" value="1920" />
        <param name="image_height" value="1080" />
    </node>
</launch>